CommandLine/CodeConv.NetFramework/CodeConv.NetFramework.csproj

```diff
@@ -10,13 +10,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" />
```

Comment:
Func.csproj or CodeConv.csproj are using 6.0.0.

Reply:
Ah yes, I think the most correct thing is actually to downgrade CodeConv to 5.0.0 since that's what ships with the latest VS2019